modules/mongodb/mongodb_test.go

```diff
@@ -76,3 +78,38 @@ func ExampleRunContainer_connect() {
 	// Output:
 	// test
 }
+
+func TestMongoDB(t *testing.T) {
+	ctx := context.Background()
+
+	container, err := mongodb.RunContainer(ctx,
+		testcontainers.WithImage("mongo:6"),
+		mongodb.WithUsername("root"),
+		mongodb.WithPassword("password"),
+		testcontainers.WithWaitStrategy(wait.ForLog("Waiting for connections")),
+	)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	t.Cleanup(func() {
+		if err := container.Terminate(ctx); err != nil {
+			t.Fatalf("failed to terminate container: %s", err)
+		}
+	})
+
+	connStr, err := container.ConnectionString(ctx)
+	if err != nil {
+		panic(err)
```

Comment:
Given this is a regular test, not a testable example, let's use the testing library to perform the assertions.
```suggestion
		t.Fatal(err)
```